From b44f9b961315d98785198e193cba184b51d63cc4 Mon Sep 17 00:00:00 2001
From: Ken <ken@radxa.com>
Date: Fri, 5 Nov 2021 17:33:06 +0800
Subject: [PATCH] arm64: dts: radxa zero: add user led

Radxa Zero V1.4/V1.5 add user led. Flashing LED shows running kernel.

Signed-off-by: Ken <ken@radxa.com>
---
 arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts b/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
index 4b2405cdd7e83..b3d3232f86ce8 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
@@ -204,6 +204,17 @@
 		clock-frequency = <32768>;
 		pwms = <&pwm_ef 0 30518 0>; /* PWM_E at 32.768KHz */
 	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led-green {
+			label = "radxa-zero:green";
+			gpios = <&gpio_ao GPIOAO_8 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "heartbeat";
+			default-state = "on";
+		};
+	};
 };
 
 &arb {
